strapi/strapi
# [Marketplace] Add submit plugin button

### What does it do?

Adds the submit plugin button to the marketplace page

### Why is it needed?

Makes it easier to submit a plugin

### How to test it?

Go to the marketplace page and click the button it should open [market.strapi.io/submit-plugin](https://market.strapi.io/submit-plugin) in a new tab
